content/syllabuses/java-part-2.md

```diff
@@ -3,7 +3,7 @@ _db_id: 57
 title: Java Systems Development - part 2
 ---
 
-- {{% contentlink path="workshops/general-coding-best-practices" %}}
+- {{% contentlink path="topics/clean-code" %}}
```

Comment:
Why is this mentioned in part 1 and part 2? - {{% contentlink path="topics/clean-code" %}}

Reply:
why remove general-coding-best practices?